### PR TITLE
Allow registration for 10-12 year olds on physics

### PR DESCRIPTION
**Pull request by chrisjpurdy**
I've used placeholder text in the "ask your parents for consent" pop-up. 

There is another card about the proper wording of this pop-up, so this PR will have to wait for that to be figured out and copy-edited before being merged.

**It would still be useful for this to be reviewed**, so that the only thing left to do is work out the wording of the consent box.

### CODE REVIEW

File: src/app/components/pages/Registration.tsx [hunk 1]
```diff
@@ -237,31 +244,37 @@ export const Registration = withRouter(({location}:  RouteComponentProps<{}, {},
...
                                             required
                                             label="I am at least 13 years old"
-                                            disabled={registrationUser.dateOfBirth}
-                                            // TODO: Look at DateInput null vs undefined for updating DoB and maybe
-                                            // change this in future
-                                            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                                                setDobCheckboxChecked(!dobCheckboxChecked);
-                                            }}
+                                            disabled={(isPhy && dob10To12CheckboxChecked) || registrationUser.dateOfBirth}
+                                            // TODO: Look at DateInput null vs undefined for updating DoB and maybe change this in future
```
Comment (mwtrew) [inline src/app/components/pages/Registration.tsx:266]: I'm not quite sure what this todo is suggesting, can we clarify (or ideally address/remove) it?

File: src/app/components/pages/Registration.tsx [hunk 2]
```diff
@@ -278,12 +291,31 @@ export const Registration = withRouter(({location}:  RouteComponentProps<{}, {},
...
                             </h4>
                         </Col>
                     </Row>
+
+                    {/* 10-12 parental consent box */}
+                    {isPhy && confirmedTenToTwelve && <Alert color={"warning"}>
+                        <h2>Attention young'un</h2>
+                        <hr/>
+                        <p>
+                            You're under 13 and you wanna use Isaac Phy? You better make sure that you ask your mum or dad or guardian or whoever because there's some really scary stuff on this platform that they might want to know about beforehand.
```
Comment (mwtrew) [inline src/app/components/pages/Registration.tsx:306]: I don't think we need to wait for text from the content team when this is already perfect.

**Review by mwtrew — APPROVED** [review]

File: src/app/components/pages/Registration.tsx (hunk 1 above)
Comment (chrisjpurdy) [reply src/app/components/pages/Registration.tsx:266]: I had no idea either, I'll git blame it!

Comment (chrisjpurdy) [reply src/app/components/pages/Registration.tsx:266]: Yeah it can almost certainly be safely removed

Comment (mwtrew) [reply src/app/components/pages/Registration.tsx]: Ah - I thought you added it there, my mistake. It's good to get these things tidied up regardless!

**Pull request merged**